### PR TITLE
[Slider] Deflake unit tests.

### PR DESCRIPTION
**Pull request by romoore**
Several unit tests were comparing the outcome of arithemtic by using
`XCTAssertEqual`, which requires absolute precision for floating point numbers.
Due to rounding errors, this is neither important nor desirable. Instead,
`XCTAssertEqualWithAccuracy` should be used.

Also corrected the order of the assertion parameters. The test value should
come before the expected value because the failure message has the template,
@"(%@) is not equal to (%@)" and when read it implies that the second value
(sentence object) is the correct value.

Removed some debug messages that managed to creep in there a while back.

### CODE REVIEW

File: components/Slider/tests/unit/SliderTests.m [hunk 1]
```diff
@@ -1103,7 +1098,7 @@ - (void)testAccessibilityIncrementDiscreteSlider {
     CGFloat stepValue = (self.slider.maximumValue - self.slider.minimumValue) /
         (self.slider.numberOfDiscreteValues - 1);
     CGFloat expectedValue = self.slider.minimumValue + stepValue;
-    XCTAssertEqualWithAccuracy(self.slider.value, expectedValue, 0.0001,
+    XCTAssertEqualWithAccuracy(self.slider.value, expectedValue, kEpsilonAccuracy,
```
Comment (codeman7) [inline components/Slider/tests/unit/SliderTests.m:1101]: Why the change from `0.001` to `kEpsilonAccuracy`, I thought `0.001` was okay in tests?

**Review by codeman7 — APPROVED** [review]

File: components/Slider/tests/unit/SliderTests.m (hunk 1 above)
Comment (romoore) [reply components/Slider/tests/unit/SliderTests.m:1101]: I was just reusing the accuracy value that was already in-use within the file. It didn't seem like the difference (0.0001 to 0.001) would have any effect on the calculations being performed.
